docs/runbooks/PromscalePostgreSQLSharedBuffersLow.md

```diff
@@ -0,0 +1,20 @@
+# PromscalePostgreSQLSharedBuffersLow
+
+## Meaning
+
+Open chunks (the chunks where data is current written into) can't fit into
+PostgreSQL shared buffers. Total size is calculated by summing up all chunk
+relations and indexes sizes.
+
+## Impact
+
+Database performance might suffer. In particular the ingest speed might get affected.
```

Comment:
I am happy to change it. What would be your formulation. This alert does fall into `unknown` territory a bit. It's hard to exactly measure the impact since it depends on many things: hardware setup, load,...

Reply:
The `Impact` section is not about dependencies, it is about things that happen to the monitored application because something is triggering this alert (as in "ingest is not keeping up with the load"). The `Diagnosis` section (which for unknown to me reasons is missing here) is about listing potential things that need to be checked (hardware setup, load, etc.).

In short, when someone wakes up at 2 am due to this alert firing, sections should tell that person what is going on and how to fix this.
`Meaning` should tell what this is about
`Impact`  should tell what is affected
`Diagnosis` should tell how to debug this thing and drill down into the problem
`Mitigation` should list steps to prevent this alert from firing. Basically how to solve issues that can be discovered in `Diagnosis`.